src/widget.cpp

```diff
@@ -108,6 +108,10 @@ void SetupWidgetDimensions()
 	WidgetDimensions::scaled.hsep_normal  = ScaleGUITrad(WidgetDimensions::unscaled.hsep_normal);
 	WidgetDimensions::scaled.hsep_wide    = ScaleGUITrad(WidgetDimensions::unscaled.hsep_wide);
 	WidgetDimensions::scaled.hsep_indent  = ScaleGUITrad(WidgetDimensions::unscaled.hsep_indent);
+
+	WidgetDimensions::scaled.toolbar_spacer = ScaleGUITrad(WidgetDimensions::unscaled.toolbar_spacer);
```

Comment:
Do these scaled dimensions actually get used anywhere? Everything I've seen in this PR is unscaled, although I could have glazed over and missed one.